workflow/artifacts/common/load_to_stream_test.go

```diff
@@ -115,7 +116,7 @@ func TestLoadToStream(t *testing.T) {
 				}
```

Comment:
can the `panic` up here also be replaced by a `require.NoError`? there's a couple `panic`s in the tests that seem like they should perhaps be a `require.NoError` from a quick glance